test/unit_test/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-add_executable(vw-unit-test.out main.cc stable_unique_tests.cc)
+add_executable(vw-unit-test.out main.cc cb_explore_adf_test.cc explore_test.cc stable_unique_tests.cc)

```

Comment:
Is there a reason to use extra tests of this sort rather than RunTests?  RunTests seems somehow cleaner since you don't need to recompile.

Reply:
I believe this sort of code issue is best tested using a unit test, especially since it might not repro with command line and stdin. I think we should aim to add a test/unit test for each bug as we fix them. Helps reduce the risk of regression and builds up the testing suite over time